mam/examples/ntru-key-exchange.c

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright (c) 2019 IOTA Stiftung
+ * https:github.com/iotaledger/entangled
+ *
+ * MAM is based on an original implementation & specification by apmi.bsu.by
+ * [ITSec Lab]
+ *
+ * Refer to the LICENSE file for licensing information
+ */
+
+#include <stdio.h>
+#include "mam/api/api.h"
+#include "mam/ntru/ntru.h"
+
+#define SENDERSEED "SENDERSEED9SENDERSEED9SENDERSEED9SENDERSEED9SENDERSEED9SENDERSEED9SENDERSEED99999"
+#define RECEIVERSEED "RECEIVERSEED9RECEIVERSEED9RECEIVERSEED9RECEIVERSEED9RECEIVERSEED9RECEIVERSEED9999"
+#define NONCE "ABCDEFGHIJKLMNOPQRSTUVWXYZABCD"
```

Comment:
Should the nonce be only 10 trytes?

Reply:
it has to be 27 trytes

Reply:
can you please remove the extra trytes for clarity?

Reply:
Done. Nonce is now 9 trytes/27 trits in size.